wallet/wallet.go

```diff
@@ -157,10 +157,13 @@ func (wal *Wallet) GetBlockExplorerURL(txnHash string) string {
 }
 
 //GetUSDExchangeValues gets the exchange rate of DCR - USDT from a specified endpoint
-func (wal *Wallet) GetUSDExchangeValues(target interface{}) {
+func (wal *Wallet) GetUSDExchangeValues(target interface{}, errChan chan error) {
```

Comment:
You don't need to pass an error channel to this method. You can just call the method in a go routine on the page and handle the error there.

Reply:
The reason for passing the error channel is that the said function  is called while initializing the page. https://github.com/planetdecred/godcr/pull/320/files#diff-0781f5f507ec841c7b2be53ba215618b87736d69def70b1c96389b9331a32893R251.
the error returned from the function cannot be handle on load of the page as it returns an error 
<img width="469" alt="image" src="https://user-images.githubusercontent.com/27733432/111793672-c762ae00-88c5-11eb-82c8-6a6231d4c668.png">

Reply:
I'm not sure what to make of the link. I've had a look at the code and the approach I suggested will work.